lib/features/support/hooks.rb

```diff
@@ -161,11 +143,26 @@
   # when a test fixture starts (which can be before the first Before scenario hook fires).
   Maze::Server.errors.clear
   Maze::Server.sessions.clear
+  Maze::Server.builds.clear
   Maze::Server.invalid_requests.clear
   Maze::Runner.environment.clear
   Maze::Store.values.clear
 end
 
+def output_failed_requests(request_type)
```

Comment:
Nice improvement factoring it into a method, but not that all received requests are now logged in v4, not just the failed ones.